.github/workflows/release-project-ios-spm.yml

```diff
@@ -32,9 +32,17 @@ jobs:
           TAG: ${{ github.event.inputs.version }}
       with:
         script: |
-            github.git.createRef({
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              ref: "refs/tags/${{ github.event.inputs.version }}",
-              sha: context.sha
-            })
\ No newline at end of file
+         github.rest.git.createRef({
+          owner: context.repo.owner,
+          repo: context.repo.repo,
+          ref: 'refs/tags/v${{ github.event.inputs.version }}',
```

Comment:
Should this be `outputs.version`?  Asking cause that's what I see from the macOS file: https://github.com/Vonage/client-sdk-video-macos/actions/runs/13443284691/workflow#L38